libs/librepcb/libraryeditor/newelementwizard/newelementwizardpage_entermetadata.cpp

```diff
@@ -184,14 +191,21 @@ void NewElementWizardPage_EnterMetadata::updateCategoryTreeLabel() noexcept {
           *mUi->lblCategoryTree);
       builder.setHighlightLastLine(true);
       builder.setOneLine(true);
-      builder.updateText(mContext.mElementCategoryUuid);
+      builder.updateText(mContext.mElementRootCategoryUuid);
       break;
     }
     default: {
       mUi->lblCategoryTree->setText(tr("Root category"));
       break;
     }
   }
+  if (mContext.mElementCategoriesUuid.count() > 1) {
+    mUi->lblMoreCategories->setText(
+        tr(" ... and %1 more.")
+            .arg(mContext.mElementCategoriesUuid.count() - 1));
```

Comment:
As a translator I'd consider leading or trailing spaces in a string as a mistake, therefore I usually move them to outside the tr():

```suggestion
        " " % tr("... and %1 more.")
                  .arg(mContext.mElementCategoriesUuid.count() - 1));
```

Reply:
This does seem more like an oversight on my part. The label is right-aligned and does not need any trailing or leading spaces.